content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-policy-examples.md

```diff
@@ -145,70 +142,123 @@ metadata:
   name: image-is-signed-by-github-actions
 spec:
   images:
-  # This is the release v0.3.0
-  - glob: "gcr.io/projectsigstore/policy-webhook@sha256:d1e7af59381793687db4673277005276eb73a06cf555503138dd18eaa1ca47d6"
+    # This is the release v0.3.0
+    - glob: "gcr.io/projectsigstore/policy-webhook@sha256:d1e7af59381793687db4673277005276eb73a06cf555503138dd18eaa1ca47d6"
   authorities:
-  - keyless:
-      # Signed by Fulcio
-      url: https://fulcio.sigstore.dev
-      identities:
-      # Matches the Github Actions OIDC issuer
-      - issuer: https://token.actions.githubusercontent.com
-        # Matches a specific GitHub workflow on main branch. Here we use the
-        # Sigstore policy controller example testing workflow as an example.
-        subject: "https://github.com/sigstore/policy-controller/.github/workflows/release.yaml@refs/tags/v0.3.0"
+    - keyless:
+        # Signed by Fulcio
+        url: https://fulcio.sigstore.dev
+        identities:
+          # Matches the Github Actions OIDC issuer
+          - issuer: https://token.actions.githubusercontent.com
+            # Matches a specific GitHub workflow on main branch. Here we use the
+            # Sigstore policy controller example testing workflow as an example.
+            subject: "https://github.com/sigstore/policy-controller/.github/workflows/release.yaml@refs/tags/v0.3.0"
 ```
 
-## Policy allowing all images
+## Policy allowing GKE images
```

Comment:
Should this include "trusted" in the heading?

Reply:
Tagging @pdeslaur to see if he has thoughts — I don't have a strong opinion either way!